rollingpin/elasticsearch.py

```diff
@@ -0,0 +1,48 @@
+import time
+import json
+
+from twisted.internet import reactor
+from twisted.web.client import Agent
+from twisted.web.http_headers import Headers
+from twisted.web.test.test_newclient import StringProducer
+
+
+class ElasticSearchNotifier(object):
+    def __init__(self, config, components):
+        base_url = config["elasticsearch"]["endpoint"]
+        index, type = config["elasticsearch"]["index"], config["elasticsearch"]["type"]
+        self.endpoint = "%s/%s/%s/_bulk" % (base_url, index, type)
+        self.components = components
+
+    @staticmethod
+    def bulk_updates(components):
+        now = int(time.time())
+        actions = []
+        for service in components:
+            actions.append(json.dumps({'index': {}}))
+            actions.append(json.dumps({
+                '@timestamp': now,
+                'service': service
+            }))
+        return actions
+
+    @staticmethod
+    def request_body(components):
+        return "\n".join(ElasticSearchNotifier.bulk_updates(components)) + "\n"
```

Comment:
maybe a quick comment here explaining the format of the body would be useful? i assume it's newline delimited json blobs?